content/applications/sales/crm/performance/win_loss.rst

```diff
@@ -1,26 +1,667 @@
+
+        - Adding :guilabel:`Salesperson` or :guilabel:`Sales Team` breaks up the total count of
+          leads in each :guilabel:`Stage` by who they were assigned to.
+        - Adding :guilabel:`Medium` or :guilabel:`Source` can reveal what marketing avenues generate
+          more sales.
+      - **Add Custom Group > Active**: Clicking :menuselection:`Add Custom Group --> Active`
+        separates the results into Won (:guilabel:`true`) or Lost (:guilabel:`false`). This shows
+        at what stage leads are being marked won or lost.
+
+      .. image:: win_loss/search-panel-filters-and-group-by-options.png
+         :align: center
+         :alt: The Search menu open and the Won and Lost filters highlighted.
+
+   .. tab:: Pivot View
+
+      By default, :guilabel:`Pivot View` groups win/loss reports by :guilabel:`Stage` and measures
+      :guilabel:`Expected Revenue` instead of the :guilabel:`Count` of leads.
+
+      To flesh out the table, click the :guilabel:`down arrow` next to the search bar, and in the
+      pop-up menu, replace the :guilabel:`Stage` grouping with something like
+      :guilabel:`Salesperson` or :guilabel:`Medium`.
+
+      Then click the :guilabel:`Measures` button and add the :guilabel:`Count` of leads back into
+      the report.
+
+      Other useful measures for :guilabel:`Pivot View` include :guilabel:`Days to Assign` and
+      :guilabel:`Days to Close`.
+
+      .. image:: win_loss/win-loss-pivot-view.png
+         :align: center
+         :alt: A win/loss report in Pivot View displays the data in table form.
+
+      .. important::
+         By default, the :guilabel:`Insert In Spreadsheet` button may be greyed out due to the
+         report containing :guilabel:`duplicate group bys`.
+
+         To fix this, replace the :guilabel:`Stage` grouping in the search bar with another option.
+
+   .. tab:: List View
+
+      In :guilabel:`List View`, a win/loss report displays all leads on a single page.
+
+      To add more columns to the list, click the :guilabel:`Filters` icon in the top-right of the
+      page, and select options from the resulting drop-down menu. Some useful filters include:
+
+      - **Campaign**: Shows the marketing campaign that originated each lead.
+      - **Medium**: Shows the marketing medium (Banner, Direct, Email, Google Adwords, Phone,
+        Website, etc.) that originated each lead.
+      - **Source**: Shows the source of each lead (Newsletter, Lead Recall, Search Engine, etc.).
+
+      To better organize the list, click the :guilabel:`down arrow` next to the search bar, and
+      add more relevant groupings or re-organize the existing ones.
+
+      To re-order the nesting, remove all :guilabel:`Group By` options and re-add them in the
+      desired order.
+
+      .. image:: win_loss/win-loss-list-view.png
+         :align: center
+         :alt: A win/loss report in List View displays all leads in an easy-to-read list.
```

Comment:
This section seems repetitive, and I feel like all of this was adequately explained above. I'd suggest removing it.